prijateli_tree/app/main.py

```diff
@@ -25,7 +25,7 @@
 Base.metadata.create_all(bind=engine)
 
 
-app = FastAPI()
+app = FastAPI(debug=True)
```

Comment:
We can use a configuration flag for this, look at the config object and the environment variables in the docker file.